src/malcolm/malcolmSocketHandler.js

```diff
@@ -16,14 +26,32 @@ const configureMalcolmSocketHandlers = (inputSocketContainer, store) => {
 
   socketContainer.socket.onopen = () => {
     console.log('connected to socket');
+    let path = `${store.getState().malcolm.navigation}`;
```

Comment:
This is changing shape and used by the nav bar component, you actually want to use `path=store.getState().router.location.pathname` I think, and then you don't have to do all this splitting and joining